pyverbs/device.pyx

```diff
@@ -1078,7 +1078,8 @@ def width_to_str(width):
 
 def speed_to_str(speed):
     l = {0: '0.0 Gbps', 1: '2.5 Gbps', 2: '5.0 Gbps', 4: '5.0 Gbps',
-         8: '10.0 Gbps', 16: '14.0 Gbps', 32: '25.0 Gbps', 64: '50.0 Gbps'}
+         8: '10.0 Gbps', 16: '14.0 Gbps', 32: '25.0 Gbps', 64: '50.0 Gbps',
+         128: '100.0 Gbps'}
```

Comment:
You will need to add new value in devinfo.c also

Reply:
> You will need to add new value in devinfo.c also

I believe that this change should come with the libibverbs update (updating the rates of ibv_devinfo which should come soon - not by me).
I guess I could have waited with this PR, but didn't want it to be missed out.